frontend/components/side_panels/HostSidePanel/PanelGroupItem/PanelGroupItem.jsx

```diff
@@ -56,7 +79,10 @@ class PanelGroupItem extends Component {
     return (
       <button className={wrapperClassName} onClick={onLabelClick}>
         <div className={`${baseClass}__flexy`}>
-          <span className={`${baseClass}__name`}>{displayText}</span>
+          <span className={`${baseClass}__name`}>
+            {label_type === "builtin" && displayIcon(name)}&nbsp;
```

Comment:
I know @gillespi314 and I prefer `{" "}` but it might be good to specify not to wrap between icon and a text with `&nbsp;`? Idk @zwass -- any overarching opinion?

Reply:
I wasn't familiar with this so had a quick look. It makes sense to me to use a "non-breaking space" in a place we don't want any wrapping, such as between the icon and text like you mention.

Unless I'm missing something, my preference would be to use `&nbsp;` where the non-breaking behavior is desired and `{" "}` where breaking is okay.